Classes/Controller/BackendModuleController.php

```diff
@@ -127,19 +156,22 @@ public function showAvailableDimensionsAction($siteNodeName)
      */
     public function showJobPostingsForSiteAction($siteNodeName, $siteDimension)
     {
+        $flashMessages = $this->controllerContext->getFlashMessageContainer()->getMessagesAndFlush();
+
         $selectedRootNode = $this->getSelectedRootNodeByDimension($siteNodeName, $siteDimension);
         $query = new FlowQuery([$selectedRootNode]);
         $jobPostings = $query->find('[instanceof TechDivision.Jobs:Document.JobPosting]')->get();
 
         $this->view->assignMultiple(array(
```

Comment:
Pls use only one array notation, i.e. "[]" as in Line 162

Reply:
*Updated*